Submitted By: Andrew "Weibullguy" Rowland <darowland_at_ieee_dot_org>
Date: 2007-10-23
Initial Package Version: 0.16.17
Origin: Fedora Project
Description: Suppresses elksemu
Upstream Status: Unknown

--- dev86-0.16.17/makefile.in.noelks	2006-01-25 17:03:02.000000000 -0500
+++ dev86-0.16.17/makefile.in	2006-01-25 17:03:19.000000000 -0500
@@ -89,10 +89,10 @@
 
 #ifdef GNUMAKE
 all: check_config bcc86 cpp unproto copt as86 ar86 ld86 objdump86 \
-     library lib-bsd alt-libs elksemu
+     library lib-bsd alt-libs 
 
 install: check_config install-bcc install-man \
-     install-lib install-emu
+     install-lib
 
 install-all: install install-other
 
